lib/board.rb

```diff
@@ -56,6 +56,17 @@ def play(move)
         @cells[move-1] = current_marker
     end
 
+    def to_s()
```

Comment:
I would leave out the `()` if the method has no parameters

Reply:
`to_s` is an existing Ruby method. Did you consciously make the choice to overwrite it here? Why?